README.md

```diff
@@ -230,21 +230,16 @@ about using `dds`.
 
 # Helper Commands
 
-Executing PMM in script mode provides some additional helper commands to work
-with your project.
-
-> NOTE: This will create a local copy of the PMM code in a `_pmm` directory. It
-> is safe to delete or `.gitignore` this directory.
+After PMM has run for the first time, it will generate a sh and bat script that can be used to maintain your PMM installation
```

Comment:
I'd leave docs for both options for running the `pmm.cmake` script.

```suggestion
The `pmm.cmake` script can be run with `-P` to access a set of utility subcommands and options. See `cmake -P pmm.cmake /Help` for options. Additionally, after PMM has run for the first time, it will generate a sh and bat script that can be used to access the same set of options without needing `cmake -P pmm.cmake`:
```